<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ChangePassword</class>
  <widget class="QDialog" name="ChangePassword" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>331</width>
        <height>149</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Change Password</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <layout class="QGridLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item row="2" column="1" >
            <widget class="QLineEdit" name="le_pwver" >
              <property name="echoMode" >
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
          <item row="0" column="1" >
            <widget class="QLineEdit" name="le_pwcur" >
              <property name="echoMode" >
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
          <item row="2" column="0" >
            <widget class="QLabel" name="lb_pwver" >
              <property name="text" >
                <string>Confirm new password:</string>
              </property>
            </widget>
          </item>
          <item row="0" column="0" >
            <widget class="QLabel" name="lb_pwcur" >
              <property name="text" >
                <string>Current password:</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1" >
            <widget class="QLineEdit" name="le_pwnew" >
              <property name="echoMode" >
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
          <item row="1" column="0" >
            <widget class="QLabel" name="lb_pwnew" >
              <property name="text" >
                <string>New password:</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="Spacer2" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <widget class="Line" name="Line1" >
          <property name="frameShape" >
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="BusyWidget" name="busy" />
          </item>
          <item>
            <spacer name="Spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>110</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="IconButton" name="pb_close" >
              <property name="text" >
                <string>&amp;Close</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="IconButton" name="pb_apply" >
              <property name="text" >
                <string>&amp;Apply</string>
              </property>
              <property name="psiIconName" stdset="0" >
                <string>psi/register</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>le_pwcur</tabstop>
    <tabstop>le_pwnew</tabstop>
    <tabstop>le_pwver</tabstop>
  </tabstops>
</ui>
